blocks/lead-art-block/features/leadart/default.jsx

```diff
@@ -65,12 +65,20 @@ class LeadArt extends Component {
     } = this.state;
 
     const { arcSite, customFields, id } = this.props;
-    const { hideTitle = false, hideCaption = false, hideCredits = false } = customFields;
+    const {
+      hideTitle = false,
+      hideCaption = false,
+      hideCredits = false,
+      imageLoadingStrategy = 'eager',
+    } = customFields;
 
     let AdBlock;
 
+    // todo: refactor dynamic loading of another block
     try {
+      /* istanbul ignore next */
```

Comment:
ignored these lines as testing them is going to require looking at implementation details of installed packages